### ISSUE TITLE
Passing number to options will not replace the placeholder

### ISSUE THREAD

**Issue by jankapunkt** [VeliovGroup/Meteor-Internationalization#10]
Reproduction:

- define i18n.json with a simple replacer pattern:

```json
{
  "max": "You reached {{max}} %"
}
```

Then pass a number value to it:

```js
const max = 75
i18n.get('max', { max }) // returns You reached {{max}} %
```

If the value is a string, it's all fine:

```js
const max = '75'
i18n.get('max', { max }) // returns You reached 75 %
```

**Comment by dr-dimitru** [comment]
@jankapunkt solved https://github.com/veliovgroup/Meteor-Internationalization/commit/26703036935d8f413e698677a2964d37affb2bcc
Thank you for discovering this one 🙏

**Comment by dr-dimitru** [comment]
@jankapunkt fixed in [📦 v3.2.1](https://github.com/veliovgroup/Meteor-Internationalization/releases/tag/3.2.1)
